apps/io-func-wallet-solution-backend/openapi.yaml

```
@@ -35,6 +35,31 @@ paths:
         "500":
           $ref: "#/components/responses/Unexpected"
 
+  /wallet-instance:
+    post:
+      operationId: createWalletInstance
+      summary: Create a wallet instance
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                challenge:
+                  type: string
+                key_attestation:
+                  type: string
+                hardware_key_tag:
+                  type: string
```

What do you think about adding the format to indicate that it's a base64?

```suggestion
                challenge:
                  type: string
                  format: binary
                key_attestation:
                  type: string
                  format: binary
                hardware_key_tag:
                  type: string
```